src/settings/index.js

```diff
@@ -1,8 +1,11 @@
 import React from 'react';
-import { FormattedMessage } from 'react-intl';
+import { FormattedMessage, injectIntl } from 'react-intl';
 
 import { Settings } from '@folio/stripes/smart-components';
 
+import { TitleManager } from '@folio/stripes-core';
+import { stripesShape } from '@folio/stripes/core';
```

Comment:
```
import { TitleManager, stripesShape } from '@folio/stripes/core`;
```
Note that's `stripes/core` so you're importing via the `@folio/stripes` wrapper package.

Reply:
Done!